packages/backend/src/util/parsePublishableKey.ts

```diff
@@ -13,7 +13,7 @@ export function buildPublishableKey(frontendApi: string): string {
   return `${keyPrefix}${btoa(`${frontendApi}$`)}`;
 }
 
-export function parsePublishableKey(key: string): PublishableKey | null {
+export function parsePublishableKey(key: string | undefined): PublishableKey | null {
```

Comment:
❓ why did we allowed the `undefined` here? Was there a bug reported for legacy (using frontendApi) customers ?

Reply:
@dimkl  I made this change because I wanted to maintain the consistency between this function and the same one written inside `packages/shared/src/utils/keys.ts`.